p2p/peer_error.go

```diff
@@ -73,6 +73,7 @@ const (
 	DiscSelf
 	DiscReadTimeout
 	DiscSubprotocolError
+	DiscThrottled
```

Comment:
You cannot add reasons to this list because these values are part of the protocol definition.